lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -280,6 +280,25 @@ bool removeExcludeFromRendering(const Ufe::Path& ufePulledPath)
     return true;
 }
 
+//------------------------------------------------------------------------------
+//
+// Turn on the mesh flag to allow topological modifications.
+bool allowTopologyModifications(MDagPath& root)
+{
+    MDGModifier& dgMod = MDGModifierUndoItem::create("Allow topology modifications");
+
+    MItDag dagIt;
+    dagIt.reset(root, MItDag::kDepthFirst, MFn::kMesh);
```

Comment:
Seems like a shame that LockNodesUndoItem does a traversal, then we do another traversal here.